php/libraries/NDB_BVL_Instrument.class.inc

```diff
@@ -54,59 +38,59 @@ abstract class NDB_BVL_Instrument extends NDB_Page
     /**
      * Test name (short name), equivalent to test_names.Test_name
      *
-     * @access private
+     * @access public
      */
-    var $testName;
+    public $testName;
 
     /**
      * Instrument instance CommentID
      *
-     * @access private
+     * @access public
      */
-    var $commentID;
+    public $commentID;
 
     /**
      * Database table containing data referenced by $this->commentID
      *
-     * @access private
+     * @access protected
      */
-    var $table;
+    protected $table;
 
     /**
      * Cache location for the date of administration
      *
-     * @access private
+     * @access public
      */
-    var $dateOfAdministration;
+    public $dateOfAdministration;
 
     /**
      * Additional form defaults - should only be used for lorisform
      * static elements!
      *
-     * @access private
+     * @access public
      */
-    var $localDefaults = ['emptyCell' => '&nbsp;'];
+    public $localDefaults = ['emptyCell' => '&nbsp;'];
 
     /**
      * String to separate the group elements
      *
-     * @access private
+     * @access public
      */
-    var $_GUIDelimiter = "</td>\n<td>";
+    public $GUIDelimiter = "</td>\n<td>";
 
     /**
      * Commonly used level of indentation
      *
-     * @access private
+     * @access public
      */
-    var $indent = "&nbsp;&nbsp;&nbsp;&nbsp;";
+    public $indent = "&nbsp;&nbsp;&nbsp;&nbsp;";
 
     /**
      * Array of required fields to check
      *
-     * @access private
+     * @access protected
      */
-    var $_requiredElements = [];
+    protected $requiredElements = [];
```

Comment:
This is changing the variable name which is going to break any instrument subclass that uses it.

Reply:
PHPCS has a rule where only privately scoped variable can be prefixed with an underscore. We can change this to private or disable phpcs. Which do you want?

Reply:
Keep it `var` until there's a better solution, either changing the name or changing the scope is going to break things and we shouldn't be disabling phpcs on anything.